server/src/routes/songRoutes.js

```diff
@@ -0,0 +1,14 @@
+const Router = require("express").Router;
+// const { authMiddleware } = require("../middleware");
+const { songController } = require("../controllers");
+
+const songRouter = Router();
+
+//http://localhost:4000/songs/
+songRouter.get("/all", songController.fetchSongs);
+songRouter.get("/:id", songController.getSongById);
+songRouter.patch("/:id", songController.updateSong);
```

Comment:
Please, insert the authMiddleware

Reply:
On my way to fix this! Thank you for the review!